### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by GhostEsso**
- I added any missing JSX elements to your project (without adding extra functionality),

- My  final design match Zeplin's design,

- I used the CSS rules detailed in Zeplin to style the website.

-UI 
image

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -154,3 +149,35 @@ h2 {
...
+
+.category-input {
+  width: 15%;
+}
+
+.update-progress-button {
+  background-color: #3498db;
+  color: #fff;
+  border: none;
+  padding: 8px 16px;
```
Comment (V-Blaze) [inline src/App.css]: - You have done great so far :+1: :+1: But there are few requirements you need to meet before moving to the next milestone. :+1:  :100: 
> Your final design should match Zeplin's design (this includes all elements + the same styling)

- [ ] As stated in the project requirement, your design must match the given design, currently, your design does not have the progress bar that's on the design, I'll suggest you implement this too. :smile: :wave: 

- - Your design:

Screenshot from 2023-08-24 18-28-05

- - Zeplin design:
Selection_010

**Review by V-Blaze — CHANGES_REQUESTED** [review]
## Hi @GhostEsso 

## Required changes 🖊️ 
# Almost there 🟡 🟦 🟨 

giphy


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing the project requirements, but there are still some issues I think you need to improve on. 😄 👋 


 ## Highlights
- Clean and organized code✔
-  Correct git flow👋 
-  Linters setup correctly ✔

 ## Required Changes ♻️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.css (hunk 1 above)
Comment (GhostEsso) [reply src/App.css]: @V-Blaze 
image

**Review by Felix45 — APPROVED** [review]
# Approved :green_circle: 
Hi @GhostEsso, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights ✔️ 
- [x] Good UI design ✅ 
- [x] Application is functioning as expected ✅ 
- [x] Git flow is used correctly ✅  
- [x] Pull request has a descriptive title and summary ✅ 
- [x] Repository has a professional readme file ✅    
- [x] Implemented all the changes from the previous reviewer ✅  

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**